cmd/probe/smartprobe.go

```diff
@@ -34,7 +34,7 @@ type smartProbe struct {
 
 const (
 	smartConfigKey     = "smart-probe"
-	smartProbePriority = 2
+	smartProbePriority = 3
```

Comment:
should we have similar change for udevProbePriority as well from 1 to 2?

Reply:
ok.. seaChestProbePriority is set to 2.. so, udevProbePriority should be fine then